2/2.1.md

```diff
@@ -41,7 +41,7 @@ Prior to build via the IDE, we would have to prepare the working directory by co
 configured location. You can achive this by runnning the below command in the working directory:
 
 ```bash
-~/projects/go$ ./bn clean cruise:prepare
+~/projects/go$ ./gradlew clean fatJar
```

Comment:
when working in intellij, one can set `-PfastBuild` to skip asset precompilations, also `prepare` is needed so as to setup the db and a few other config bits so we can run `DevelopmentServer` without hiccups.

So this task should be —

`./gradlew clean prepare fatJar -PfastBuild`